package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "mgclient",
+  "version": "0.0.1",
+  "description": "Memgraph db client adapter",
+  "main": "index.js",
+  "directories": {
+    "example": "examples"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/memgraph/jsmgclient.git"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Add yourself, and I think Apache 2 is the license we use across all of our packages, right @gitbuda ?

Reply:
Though if the ISC is the go-to for JS packages, I might stick to that if possible.